packages/worker/src/token/tokenOffChainData/providers/coingecko/coingeckoTokenOffChainDataProvider.ts

```diff
@@ -35,12 +35,14 @@ export class CoingeckoTokenOffChainDataProvider implements TokenOffChainDataProv
   private readonly isProPlan: boolean;
   private readonly apiKey: string;
   private readonly apiUrl: string;
+  private readonly platformId: string;
 
   constructor(configService: ConfigService, private readonly httpService: HttpService) {
     this.logger = new Logger(CoingeckoTokenOffChainDataProvider.name);
     this.isProPlan = configService.get<boolean>("tokens.coingecko.isProPlan");
     this.apiKey = configService.get<string>("tokens.coingecko.apiKey");
     this.apiUrl = this.isProPlan ? "https://pro-api.coingecko.com/api/v3" : "https://api.coingecko.com/api/v3";
+    this.platformId = configService.get<string>("COINGECKO_PLATFORM_ID", "zksync"); // Default to zksync for backward compatibility
```

Comment:
To keep things consistent, let's move usage of `COINGECKO_PLATFORM_ID` to the [config module](https://github.com/matter-labs/block-explorer/blob/main/packages/worker/src/config.ts#L96) so it's used the same way as other env variables:

```
      coingecko: {
        isProPlan: COINGECKO_IS_PRO_PLAN === "true",
        apiKey: COINGECKO_API_KEY,
        platformId: COINGECKO_PLATFORM_ID || "zksync",
      },
```
and then use it like this:
```
configService.get<string>("tokens.coingecko.platformId")
```